shared/middleware/files_api.rb

```diff
@@ -117,6 +140,28 @@ def allowed_file_type?(endpoint, extension)
   end
 
   #
+  # PUT /v3/(assets|sources)/<channel-id>?abuse_score=<abuse_score>
```

Comment:
Yeah, I wasnt too sure about this. I am modifying the entire collection, but I'm not actually replacing it. i.e. I would think in most cases if you're doing a PUT to a resource you're providing the new resource in the body. I'm providing no body, and just modifying metadata. Thoughts?